dev/ci/README.md

```diff
@@ -71,24 +67,38 @@ When you submit a pull request (PR) on Coq GitHub repository, this will
 automatically launch a battery of CI tests. The PR will not be integrated
 unless these tests pass.
 
-Currently, we have two CI platforms:
+We are currently running tests on many different platforms:
 
-- Travis is the main CI platform. It tests the compilation of Coq, of the
+- GitLab CI is the main CI platform. It tests the compilation of Coq, of the
   documentation, and of CoqIDE on Linux with several versions of OCaml /
   camlp5, and with warnings as errors; it runs the test-suite and tests the
-  compilation of several external developments. It also tests the compilation
-  of Coq on OS X.
+  compilation of several external developments.
+
+- Circle CI runs tests that are redundant with GitLab CI and will be removed
+  eventually.
+
```

Comment:
This discussion belongs elsewhere but there is still a good use case for Circle: chains of long depending contribs. There, Circle does a better job. So given that Circle maintenance overhead is quite low [using Docker] I wouldn't mind keeping some jobs like that on Circle.

Reply:
Well I guess it depends how do you read the sentence, maybe the current wording is OK.